apps/app/src/main/scala/org/lfdecentralizedtrust/splice/console/ScanAppReference.scala

```diff
@@ -752,7 +723,8 @@ final class ScanAppBackendReference(
     new ParticipantClientReference(
       spliceConsoleEnvironment,
       s"remote participant for `$name``",
-      config.participantClient.getParticipantClientConfig(),
+      // TODO this metricsRegistry stuff looks a but crazy - revisit
+      config.participantClient.getParticipantClientConfig(spliceConsoleEnvironment.environment.metricsRegistry.generateMetricsFactory(MetricsContext.Empty)),
```

Comment:
This is of course not right, it's mainly caused by participantClientConfig (config object) having a getToken method, which feels off to me, whic of courserequires a HttpClient, so I probably have to refactor that.